### PR TITLE
Fix aliasing bug with chrome

### PR DESCRIPTION
**Pull request by EarthenSky**
Weird aliasing bug with chrome, see https://stackoverflow.com/questions/74502978/object-fit-cover-gives-pixelated-images-on-chrome for more info

I thought I was going crazy hahaha

### CODE REVIEW

**Comment by micahdbak** [comment]
Whoaaa

**Comment by micahdbak** [comment]
Tell ya what, I'll look into then when i work on https://github.com/CSSS/csss-site-frontend/issues/36

**Comment by micahdbak** [comment]
wait this is a PR not an issue 🤣

File: src/main/pages/Home.js [hunk 1]
```diff
@@ -104,6 +104,7 @@ export const Home = () => {
             src="/static/files/main/president.jpg"
             alt="President Portrait"
             className="w-64 h-80 object-cover rounded-lg float-left mr-6 mb-4"
+            style="overflow-clip-margin: unset;"
```
Comment (micahdbak) [inline src/main/pages/Home.js:107]: I don't think you can do this in react FYI; the style prop takes in a JS object for styles.

Do this instead:

```js
style={{
  'overflow-clip-margin': 'unset'
}}
```

**Review by micahdbak — COMMENTED** [review]
Have you tried running your changes locally to see if this fixes the issue?

File: src/main/pages/Home.js (hunk 1 above)
Comment (micahdbak) [reply src/main/pages/Home.js:107]: I could be wrong! But the way I sent is how React wants you to do it

Comment (EarthenSky) [reply src/main/pages/Home.js:107]: ahh, I see. I was just testing in the chrome debugger

Comment (micahdbak) [reply src/main/pages/Home.js:107]: Make sure to put the double `{{` btw! It looks weird but in react props work as follows:

prop="constant value",

prop={valueThatIsDynamicJS},

prop={ /* the following is an object: */ { ... }}

hence

prop={{ ... }}

Comment (EarthenSky) [reply src/main/pages/Home.js]: I now notice that's what you've been doing for the rest of the items. Def better than raw strings 

https://github.com/CSSS/csss-site-frontend/pull/42/commits/dcb095f749c48f097c4e19f0d1620d1f7ec2d3bd

**Comment by micahdbak** [comment]
Looks good! Did you test it?

**Comment by micahdbak** [comment]
If yes I'll approve

**Comment by micahdbak** [comment]
Oh also, can you check if anywhere else in Home.js uses object-cover? Other elements might need the same style prop to fix the aliasing bug

**Comment by micahdbak** [comment]
Just tested locally, it fixes it!

**Comment by EarthenSky** [comment]
> bro what happened

Apparently, the installed version of node I had wasn't new enough to install the node version package manager? And so it got into a state where npm would fail if you ran any non-trivial command because the version of npm was up to date but node wasn't. The internet's solution was to just completely re-install node, and it seems this was somewhat common? I don't remember very well, but I believe I "gave up" around halfway though since I didn't plan on developing with react anyways...

I will hit you up tonight! :sob:

**Comment by micahdbak** [comment]
damn that sounds rough 😭

**Comment by micahdbak** [comment]
good luck soldier, hit me up if needed

**Comment by micahdbak** [comment]
ill approve and merge cuz i love 1 line long PRs

**Review by micahdbak — APPROVED** [review]

**Pull request merged**